manual/src/SUMMARY.md

```diff
@@ -0,0 +1,18 @@
+# Summary
+
+- [Introduction](README.md)
+
+# Basic Usage
+
+- [Installation](basic/installation.md)
+- [Feature Choices](basic/feature_choices.md)
+- [Connection and Operation](./basic/connection_and_operation.md)
+
+# Advanced
```

Comment:
IMO we can drop the "basic" vs "advanced" distinction. I'm a bit worried that users may potentially skip over the advanced stuff when getting started, even though ideally all users, from beginners to experts, should be using serde, for example.

Reply:
Done.